src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -268,6 +268,20 @@ internal override void ForceComplete(SourceLocation? locationOpt, Predicate<Symb
                         _state.SpinWaitComplete(CompletionPart.FinishValidatingReferencedAssemblies, cancellationToken);
                         break;
 
+                    case CompletionPart.StartInterceptorsChecks:
+                        if (_state.NotePartComplete(CompletionPart.StartInterceptorsChecks))
+                        {
+                            DiscoverInterceptors();
```

Comment:
>DiscoverInterceptors(); [](http://example.com/codeflow?start=28&length=23)

It looks like filtering is not respected.

Reply:
BTW, is ENC build broken today in presence of interceptors? CC @333fred

Reply:
@333fred 
> I don't think so. 

I think we need to do some testing in this area. The scenario I have in mind is some code is modified before or after an intercepted location, but the modification doesn't change any code produced by the interceptor, i.e. it still intercepts the same location with the same method. The question is: "Are we going to properly detect the fact of the interception during EnC even though the code produced by the interceptor doesn't change?"

Reply:
Probably worth having some tests, yeah.